deploy/kubernetes/wait-for-driver.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# This script waits for the deployments from ./deploy-driver.sh to be ready.
+
+print_usage()
+{
+    printf "wait-for-driver.sh [--windows]\n"
+    printf "\t--windows: wait on the windows deployment rather than linux"
+    echo
+}
+
+node_daemonset=csi-gce-pd-node
```

Comment:
there are some functions like waitForDaemonSets in k8s.io/kubernetes/test/e2e/.., is it good to reuse those?

Reply:
See my reply above to @msau42 on the existing driver using shell commands rather than client-go in the first place.